src/routers/meta_transaction_router.ts

```diff
@@ -34,7 +35,16 @@ export const createMetaTransactionRouter = (
      *
      * https://0x.org/docs/guides/v3-specification#executing-a-transaction
      */
-    router.post('/submit', asyncHandler(handlers.submitZeroExTransactionIfWhitelistedAsync.bind(handlers)));
+    router.post('/submit', asyncHandler(handlers.submitTransactionIfWhitelistedAsync.bind(handlers)));
```

Comment:
there is a [path in the runner](https://github.com/0xProject/0x-api/blob/master/src/runners/http_meta_transaction_service_runner.ts#L51) that makes all of these `/v0/` prefixed. So your `/v1` paths will actually be `/v0/v1`

Reply:
gol dang